lib/src/main/kotlin/trees/abstractTree.kt

```diff
@@ -0,0 +1,157 @@
+package trees
+
+import nodes.abstractNode
+
+abstract class abstractTree<K: Comparable<K>, V, someNode: abstractNode<K, V, someNode>> {
```

Comment:
Тоже [названия](https://kotlinlang.org/docs/coding-conventions.html#naming-rules) классов.
```suggestion
abstract class AbstractTree<K: Comparable<K>, V, someNode: abstractNode<K, V, someNode>> {
```